src/forms.py

```diff
@@ -0,0 +1,14 @@
+"""forum123's forms module."""
+
+
+from flask_wtf import FlaskForm
+from wtforms import PasswordField, StringField, SubmitField
+from wtforms.validators import DataRequired
+
+
+class RegistrationForm(FlaskForm):  # type: ignore  # pylint: disable=unused-variable
```

Comment:
We don't actually need this `# pylint: disable=unused-variable`.

`pylint` shows you this error because of `allow-global-unused-variables=no` option in `.pylintrc` file. This is my mistake I made when setting up linters.

I'll make a hotfix tommorow into `develop` branch and you will be able to remove this `# pylint: disable=unused-variable` after rebasing on last `develop` with my fix.